packages/types/src/connect/config.ts

```diff
@@ -1,5 +1,5 @@
-import { ConnectAppendError, ConnectLoginError } from './errors';
-import { ConnectLoginStates } from './states';
+import type { ConnectAppendError, ConnectLoginError } from './errors';
+import type { ConnectLoginStates } from './states';
 
 export type CorbadoConnectLoginConfig = {
   onFallback(email: string): void;
```

Comment:
onStateChange is a combination of the other handlers, right? This opens up a second way to use the component (either use the handlers or onStateChange) which makes it a bit more complex => Can we go without onStateChange or do we need it for something else?

Reply:
@incorbador currently i'm using it to detect if i should show the right side in vicroads or not depending on the open screen. Maybe i can use the handlers for that it can be a bit complexe when implementing

Reply:
Nice, thanks 👍